projects/canopy-test-app/src/app/app.component.html

```diff
@@ -567,6 +567,17 @@
               </tbody>
             </table>
 
+            <lg-heading level="3">Modal</lg-heading>
+
+            <button lgModalTrigger="modal-story" lg-button type="button" variant="outline-primary">Open modal</button>
```

Comment:
```suggestion
            <button lgModalTrigger="modal" lg-button type="button" variant="outline-primary">Open modal</button>
```
Should it match the id of the modal? (I know the test page is bust #331 )

Reply:
Done